pkg/branding/brandedtext_test.go

```diff
@@ -0,0 +1,37 @@
+package branding
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/pkg/testutils/envisolator"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetBrandedProductName(t *testing.T) {
+	envIsolator := envisolator.NewEnvIsolator(t)
+
+	tests := map[string]struct {
+		productBrandingEnv string
+		brandedProductName string
+	}{
+		"RHACS branding": {
+			productBrandingEnv: "RHACS_BRANDING",
+			brandedProductName: productBrandingNameRHACS,
+		},
+		"Stackrox branding": {
+			productBrandingEnv: "STACKROX_BRANDING",
+			brandedProductName: productBrandingNameStackrox,
+		},
+		"Default setting": {
+			productBrandingEnv: "ROX_PRODUCT_BRANDING",
+			brandedProductName: productBrandingNameStackrox,
+		},
+	}
```

Comment:
Please add a test that doesn't set the environment variable or sets its value to `""` (should be equivalent for Go) because that will be a default for cases when somebody just runs/builds without setting the variable.

Reply:
Okay, then that test will result in RHACS branding for now, but should also be changed after setting the env in CI and removing that default. I'll add a TODO there as well